tests/utils/test_helpers.py

```diff
@@ -79,3 +80,17 @@ def test_get_client_cert(config: "Config"):
     config.merge({"certificates": {"foo": {"client-cert": client_cert}}})
 
     assert get_client_cert(config, "foo") == Path(client_cert)
+
+
+def test_canonicalize_name():
+    names = ["flask", "Flask", "FLASK", "FlAsK", "fLaSk57", "flask-57"]
+    canonicalized_names = [canonicalize_name(name) for name in names]
```

Comment:
I would prefer to have it as parameterize test. IMO  that would be better to read.